pokemongo_bot/__init__.py

```diff
@@ -387,7 +387,7 @@ def _register_events(self):
         )
         self.event_manager.register_event(
             'next_egg_incubates',
-            parameters=('km_needed', 'distance_in_km',)
+            parameters=('km_needed', 'distance_in_km', 'eggs', 'eggs_inc')
```

Comment:
The call to next_egg_incubates in incubate_eggs.py only sends one parameter currently.